### PR TITLE
Fix/frontend UI typo

### PR DESCRIPTION
**Pull request by AjeyataVerma**
typo-ui

### CODE REVIEW

File: src/modules/superAdmin/organisations/OrganisationsComponent/index.jsx [hunk 1]
```diff
@@ -81,19 +89,16 @@ const OrganisationsComponent = ({
       <div className='organisations-list-section d-flex flex-column'>
         <div className='organisations-list-section-head d-flex align-items-center px-3 mb-5'>
           <h2 className='organisation-heading font-weight-bold mb-0'>
-            organisations
+            Organization
```
Comment (somnath77) [inline src/modules/superAdmin/organisations/OrganisationsComponent/index.jsx:92]: It should be plural

File: src/modules/superAdmin/organisations/OrganisationsComponent/index.jsx [hunk 2]
```diff
@@ -81,19 +89,16 @@ const OrganisationsComponent = ({
...
-            <FilterComponent count={totalOrganisations} text='Organisation' />
+            <SearchBox placeholder='Search Organization' className='mr-3' />
+            <FilterComponent count={totalOrganisations} text='Organization' />
           </div>
         </div>
-        {
-          orgListErrorMsg === '' && organisations.length === 0 ? 'Organisations Not Found' : orgListErrorMsg
-        }
+        {orgListErrorMsg === '' && organisations.length === 0
+          ? 'Organisations Not Found'
```
Comment (somnath77) [inline src/modules/superAdmin/organisations/OrganisationsComponent/index.jsx:100]: you missed `Organisations` here, line 100

**Review by somnath77 — COMMENTED** [review]
A lot of changes in this PR are irrelevant to task. 
You should push only those changes which are necessary.

File: src/modules/superAdmin/organisations/OrganisationsComponent/OrganisationComponent.jsx [hunk 3]
```diff
@@ -58,21 +58,22 @@ const OrganisationComponent = ({
...
+              //   onClickDetails={onClickDetails}
             />
             <h6 className='mb-2 font-weight-bold card-heading text-capitalize'>
-              {organisation.name}
+              {organisation.show_name}
             </h6>
 
             <p className='organisation-description text-truncate-three-line overflow-hidden mb-0'>
-              {organisation.description || `Organisation Description will be provided here.
+              {organisation.description ||
```
Comment (pooja-mane) [inline src/modules/superAdmin/organisations/OrganisationsComponent/OrganisationComponent.jsx:68]: Typo for 'Organisation'.Please correct it

File: src/modules/superAdmin/organisations/OrganisationsComponent/OrganisationComponent.jsx [hunk 4]
```diff
@@ -58,21 +58,22 @@ const OrganisationComponent = ({
...
+                `Organisation Description will be provided here.
```
Comment (pooja-mane) [inline src/modules/superAdmin/organisations/OrganisationsComponent/OrganisationComponent.jsx:69]: Typo for 'Organisation'.Please correct it. `Organisation Description will be provided here.`

File: src/modules/superAdmin/organisations/OrganisationsComponent/index.jsx (hunk 1 above)
Comment (AjeyataVerma) [reply src/modules/superAdmin/organisations/OrganisationsComponent/index.jsx:92]: corrected

File: src/modules/superAdmin/organization/OrganizationComponent/OrganizationComponent.jsx [hunk 5]
```diff
@@ -66,7 +66,7 @@ const OrganisationComponent = ({
```
Comment (somnath77) [inline src/modules/superAdmin/organization/OrganizationComponent/OrganizationComponent.jsx:66]: Line 64, if `organisation.show_name` is empty, then show `organisation.name`

File: src/modules/superAdmin/organization/OrganizationComponent/OrganizationComponent.jsx [hunk 6]
```diff
@@ -58,21 +58,22 @@ const OrganisationComponent = ({
               //   )
               // }
               onClickEdit={onClickEdit}
-            //   onClickDetails={onClickDetails}
+              //   onClickDetails={onClickDetails}
             />
             <h6 className='mb-2 font-weight-bold card-heading text-capitalize'>
-              {organisation.name}
+              {organisation.show_name}
```
Comment (somnath77) [inline src/modules/superAdmin/organization/OrganizationComponent/OrganizationComponent.jsx:64]: if `organisation.show_name` is empty, then show `organisation.name`

**Pull request closed without merging**